### DISCUSSION TITLE
Move Process and Chat columns next to the Trade ID column

### DISCUSSION THREAD

**Discussion by huey735** [bisq-network/bisq#5392] [Ideas]
As of https://github.com/bisq-network/bisq/pull/5207 we now have two new columns `Process` and `Chat` columns. These have been added to the far right of the screen. I think it makes more sense to have them next to the `Trade ID` column so it's easier to not make a mistake when closing a ticket or opening chat.

![newColumn](https://user-images.githubusercontent.com/44140017/113560748-6e03ba00-95fb-11eb-9e38-d138b2d195b9.jpg)

@leo816  @jmacxx

**Comment by ghost** [comment]
@huey735 I am ok with that and can do it no problem!  [Originally in the PR](https://github.com/bisq-network/bisq/pull/5207#issuecomment-781967447) they were on the left, @ripcurlx told me to move them to the right.  Just so everyone's on the same page and we don't repeatedly move them left-right-left-right!!  :laughing:

**Reply by huey735** [reply]
>IMO we should only have the info icon in front and move the other action icons (process and chat) to the end. It just feels weird for me having this action icons in the front (yes, I know we had the CTA buttons there as well in the past). What do you think?

@ripcurlx 

As a regular user of the tools it makes sense to have them close the Trade ID column. What do you think @leo816 ?

**Reply by leo816** [reply]
I agree, makes more sense. also the function of being able to shift through all the messages by just clicking the down arrow would be great to get back.

**Reply by ghost** [reply]
Done!

> Support agents want the process & chat buttons on the left. For users it makes sense for the chat button to be on the right, consistent with how it is shown in TradeView. Hopefully that makes everyone happy!

https://github.com/bisq-network/bisq/pull/5370#issuecomment-814967148

> also the function of being able to shift through all the messages by just clicking the down arrow would be great to get back.

Ok, I set focus to the TableView instead of filter box, up and down keys will work.  Or did you mean something else?

**Reply by ripcurlx** [reply]
Sorry for the lat reply - my intention to move them to the far right was that IMO I would expect the CTA on the right. But as you guys are working with it every day and if it prevents mistakes from your point of view I'm fine with making this change :wink:
